azure-openai-model-provider/main.py

```diff
@@ -72,15 +75,9 @@ async def chat_completions(request: Request):
     messages = data["messages"]
     messages.insert(0, {"content": system, "role": "system"})
 
-    config = await helpers.get_azure_config(data["model"])
-    if config == None:
-        raise HTTPException(status_code=400,
-                            detail="Azure config not found. Please ensure you have configured the environment variables correctly.")
-
-    client = helpers.client(
-        endpoint=config.endpoint,
-        deployment_name=config.deployment_name,
-        api_key=config.api_key
+    client = azure_client(
```

Comment:
Looks like you aren't passing `deployment_name` in here anymore, but I'm pretty sure that's required.